WMF Framework/Theme.swift

```diff
@@ -160,11 +160,11 @@ public class Theme: NSObject {
     public let blurEffectStyle: UIBlurEffectStyle
     public let keyboardAppearance: UIKeyboardAppearance
     
-    public static let light = Theme(colors: .light, preferredStatusBarStyle: .default, blurEffectStyle: .light, keyboardAppearance: .light, name: "standard")
+    public static let light = Theme(colors: .light, preferredStatusBarStyle: .default, blurEffectStyle: .light, keyboardAppearance: .light, name: "Default")
```

Comment:
A user facing name could be added as `localizedName` or what is currently `name` move to `key` and `name` could be a localized name.